### ISSUE TITLE
[Feature Request] Hide status bar in windowed mode

### ISSUE THREAD

**Issue by waitingmoon** [project64/project64#1099]
Does this emulator has a "hide status bar" option? I can't find.

**Comment by project64** [comment]
no there is not, I am sure it is not hard to do, any particular reason you do not want the status bar?

**Comment by waitingmoon** [comment]
I guess people who always need the information to be displayed in the status bar is not so much. 

In addition, the color of the status bar is brighter as compared with the game screen. It is a little eyesore during the play.

**Comment by JuggaloMAN181** [comment]
i am also wondering if there is a way to do that

**Comment by Dangie** [comment]
So, the status bar obscures the bottom segment of games being played. This interferes with games such as Banjo Kazooie.

I could play in fullscreen, however I am also playing Runescape at the same time. Thus I need to play in windowed mode so I can switch between the two games. Please add this feature or, please add a fullscreen windowed mode. 

Tl;dr The status bar actually overlaps being played in windowed mode, hampering one's experience.

Thank you for reading. (PS no I don't care about necroing an old post)

**Comment by cxd4** [comment]
>Tl;dr The status bar actually overlaps games being played in windowed mode, hampering one's experience.

That sounds like a bug in Project64-video plugin.

Generally the graphics plugin is responsible for using the spec to compute the screen rendering viewport offset with respect to hStatusBar, passed from the spec plugin info.

If you use a different video plugin you will probably not see the same issue.

**Comment by Dangie** [comment]
> If you use a different video plugin you will probably not see the same issue.

In fact you are correct. I switched from Jabo's Direct3D8 to Glide64 and it has been working perfect since.

**Comment by project64** [comment]
closing old issue, if this is still an issue it can be reopened

**Comment by magiruuvelvet** [comment]
Why was this closed? I'm currently on version 8ed4a80 and there is still no way to hide/remove the status bar for aesthetical reasons. Its just UI clutter. Not everyone wants to constantly have a status bar visible. It would be nice to hide the menu bar too to have literally just the game itself in windowed mode without any kind of UI controls. Maybe add a shortcut for that?

**Comment by IronYoshi64** [comment]
Yeah I agree. This will be a very nice thing to add. I totally hear you for your reasons.

**Comment by TheePest** [comment]
I just wanted to add this is not just a windowed issue. Even in full screen, games like Virtual Pro Wrestling 2 that don't quite reach the bottom of the screen cover up some, but not all of the status bar so you have half of a blue status bar sticking out beneath the game. Games like Zelda: Ocarina of Time in full screen do reach the bottom of the screen and cover up the bar completely.

**Comment by kaine4223** [comment]
yes pleas ehave an option to remove it. especially annoying when you're streaming.

**Comment by dftf-stu** [comment]
Can this issue not be closed -- hasn't this option since been added?

In your Project64:
- Click _Options_ then _Configuration_
- Un-tick "Hide advanced settings" then click [Apply]
- Click "Advanced" in the left-hand list
- Un-tick "Show status bar", then click [OK]

Does this not address the issue?

**Comment by kaine4223** [comment]
I still cant get rid of the status bar at the bottom.

**Comment by flagrama** [comment]
Which version?
Windowed, fullscreen, or both?
Can you screenshot that you have actually unchecked the option and post it here as well?

**Comment by dftf-stu** [comment]
@kaine4223 Please can you try these steps:

Click **Options** then **Configuration**
![STATUSB1](https://user-images.githubusercontent.com/83965343/183073147-b9855847-2187-4d11-ade1-3d4f5fe491e2.png)

Un-tick **Hide advanced settings** then click **Apply**
![STATUSB2](https://user-images.githubusercontent.com/83965343/183073183-ec77a5ff-b95b-4fce-908a-28cfbf1b895e.png)

You will then see **Advanced** appear on the left-hand side.  Click on it.
Then un-tick **Show status bar** then click **OK**
![STATUSB3](https://user-images.githubusercontent.com/83965343/183073568-2f7731eb-671e-4739-9f61-6dd84cdf3ad0.png)

The status-bar should then be gone:
![STATUSB4](https://user-images.githubusercontent.com/83965343/183073657-3b3d4f4a-5afc-41d2-8d31-c9dfd4b28f42.png)

- **If you cannot find the "Show status bar" setting:** you may be running an old version of Project64, before that setting was added.  Are you able to use a newer version?

- **If the status-bar is still visible when you are playing a game:** it might be that the video plug-in you are using has its own status-bar setting.  Go to **Options** then **Graphics setting** and have a look through the options you then see.

@project64 ; @flagrama -- seeing as a number of users have asked about this over-the-years, might I suggest putting "Hide status bar" in a more obvious place?  It does seem silly to have it hidden in "Advanced" settings, as that should be reserved for settings that come under the category of "you can change these, but doing so may cause the emulator to crash or have problems, or cause your operating-system to crash".  Hiding a status-bar is hardly on that same scale, so maybe just move it to the "General settings" page?

**Comment by project64** [comment]
rightly or wrongly I have advances setting to try to keep the experience really simple, I have the view that most app over complicate the user experience with so many options, so I was trying to keep to the very minimum options needed, with the ability to access them if you really needed to.

I know some people have suggested that all options should be visible and advanced options should not even be there.

I like the idea of the user being able to open the emu, configure there controller and then load a game and in most cases everything works.

**Comment by dftf-stu** [comment]
@project64: I'd agree with most of the current settings that are under "Advanced" being there, as clearly some of them will cause poor user-experiences if turned on.

But simply hiding a Status Bar does seem unnecessary as most other apps simply offer this on a "View" menu (e.g. Notepad or DuckStation).  _Project64_ doesn't have a View menu, so I'd suggest either putting the option at the bottom of the "Options" menu, or in the "General settings" section of the **Configuration** dialog.

There is such a thing as _over-simplifying_ a UI, and when users have to be guided step-by-step through how to find a simple setting, I'd argue that's a good example of that

**Comment by project64** [comment]
if we have 100 random users who download and use project64 to play there old favorite game, how many are going to use that feature. 

Something like configure controllers, I expect 90/100 are likely too.. at turning off status bar, I think it would be lucky that 1/100 would want to.

I would say if less then 20/100 would use that feature it most likely should be under advanced

**Comment by waitingmoon** [comment]
Well... is the status bar so important? I think the information displayed on it is unnecessary for the average user who just wants to play console games.

If we follow that idea, I think we should make it default to hidden and then add an option to 'Show status bar and additional infos'.

**Comment by project64** [comment]
@waitingmoon the most important information that is shown is confirmation that a save was made when pressing f5, for that alone I would want to keep it

**Comment by Oroborius** [comment]
2024 and still can't hide it. Unfortunate. 

And still really not possible to have borderless window for fullscreen. It's either you have a status/menu bar or you can't capture it in programs due to the exclusive fullscreen.

**Comment by Squall-Leonhart** [comment]
keep the status bar, do not add an option to hide it.

the salt industry requires it.

**Comment by project64** [comment]
In General Settings -> Advanced there is "Show Status bar", just need to uncheck it
